NthKey/View/Settings/Wallet/WalletListView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  WalletListView.swift
+//  WalletListView
+//
+//  Created by Sergey Vinogradov on 21/03/2021.
+//  Copyright © 2021 Purple Dunes. Distributed under the MIT software
+//  license, see the accompanying file LICENSE.md
+
+import SwiftUI
+
+struct WalletListView: View {
+    @ObservedObject var model: WalletListViewModel
+
+    @Binding var isShowingScanner: Bool
+    
+    @State private var walletToRemove: WalletEntity?
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            ForEach(model.items) { item in
+                WalletEntityView(item: item, selected: model.selectedWallet == item)
+                    .contentShape(Rectangle())
+                    .onTapGesture {
+                        model.selectedWallet = item
+                    }
+                    .onLongPressGesture {
+                        walletToRemove = item
+                    }
+            }
+            Button(action: {
+                isShowingScanner = true
+            }) {
+                HStack {
+                    Image(systemName: "plus.circle")
+                    Text("Add a wallet by scan QR")
```

Comment:
Let's make that: `Scan wallet QR code`

Reply:
fixed in 938f3e2ab5391df85c048c5eeb6987bc28f3c095

Reply:
Don't forget to push the commit :-)